daprdocs/content/en/developing-applications/building-blocks/state-management/howto-stateful-service.md

```diff
@@ -6,38 +6,40 @@ weight: 300
 description: "Use state management with a scaled, replicated service"
 ---
 
-In this article you'll learn how you can create a stateful service which can be horizontally scaled, using opt-in concurrency and consistency models.
+In this article, you'll learn how to create a stateful service which can be horizontally scaled, using opt-in concurrency and consistency models. Consuming the state management API frees developers from difficult state coordination, conflict resolution, and failure handling.
 
-This frees developers from difficult state coordination, conflict resolution and failure handling, and allows them instead to consume these capabilities as APIs from Dapr.
-
-## Setup a state store
+## Set up a state store
 
 A state store component represents a resource that Dapr uses to communicate with a database.
-For the purpose of this guide, we'll use a Redis state store.
-
-See a list of supported state stores [here]({{< ref supported-state-stores >}})
+For the purpose of this guide, we'll use the default Redis state store.
 
 ### Using the Dapr CLI
 
-The Dapr CLI automatically provisions a state store (Redis) and creates the relevant YAML when running your app with `dapr run`.
-To change the state store being used, replace the YAML under `/components` with the file of your choice.
+When you run `dapr init` in standalone mode, Dapr creates a default Redis `statestore.yaml` and runs a Redis state store on your local machine, located:
+
+- On Windows, under `%UserProfile%\.dapr\components\statestore.yaml`
+- On Linux/MacOS, under `~/.dapr/components/statestore.yaml`
+
+With the `statestore.yaml` component, you can easily swap out underlying components without application code changes.
+
+See a [list of supported state stores]({{< ref supported-state-stores >}}).
 
 ### Kubernetes
 
-See the instructions [here]({{<ref setup-state-store>}}) on how to setup different state stores on Kubernetes.
+See [how to setup different state stores on Kubernetes]({{<ref setup-state-store>}}).
 
-## Strong and Eventual consistency
+## Strong and eventual consistency
 
-Using strong consistency, Dapr will make sure the underlying state store returns the response once the data has been written to all replicas or received an ack from a quorum before writing or deleting state.
+Using strong consistency, Dapr will make sure the underlying state store:
 
-For get requests, Dapr will make sure the store returns the most up to date data consistently among replicas.
-The default is eventual consistency, unless specified otherwise in the request to the state API.
+- Returns the response once the data has been written to all replicas
+- Receives an ack from a quorum before writing or deleting state
 
-The following examples illustrates using strong consistency:
+For get requests, Dapr will make sure the store returns the most up-to-date data consistently among replicas. The default is eventual consistency, unless specified otherwise in the request to the state API.
```

Comment:
```suggestion
For get requests, Dapr makes sure the store returns the most up-to-date data consistently among replicas. The default is eventual consistency, unless specified otherwise in the request to the state API.
```